OpencartTest/src/com/opencart/tools/JsonDataConfig.java

```diff
@@ -0,0 +1,65 @@
+package com.opencart.tools;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class JsonDataConfig {
+
+
+    private FileReader reader;
+    private JSONArray userLoginArray;
+
+    public JsonDataConfig(String fileName) {
```

Comment:
look into jackson json serialization / desirialization . just generally easier to work with
refactor to have filereader and parser separately

Reply:
Can i make json changing in future merge?

Reply:
@ostret Can i make json changing in future merge?